README.md

```diff
@@ -18,6 +18,10 @@ Currently, the project consists of a Go microservice and a corresponding client
 
 * Unimplemented
 ```
+### Debug gRPC
+Set env variables
+`export GRPC_GO_LOG_VERBOSITY_LEVEL=99`
+`export GRPC_GO_LOG_VERBOSITY_LEVEL=info`
```

Comment:
Let's create an `.env` file for these. If they should be specific to either the client or server, then put the `.env` in their respective dirs, if not then keep it in root.

Reply:
I think I found a better solution. Will have it configurable on the server programmatically with a flag. Will push shortly.